tests/buffer_ut/buffer_ut.c

```diff
@@ -72,7 +72,13 @@ void my_gballoc_free(void* ptr)
 }
 
 #define ENABLE_MOCKS
-#include "azure_c_pal/gballoc.h"
+#include "azure_c_pal/gballoc_hl.h"
+#include "azure_c_pal/gballoc_hl_redirect.h"
+#undef ENABLE_MOCKS
+
+#include "real_gballoc_hl.h"
```

Comment:
>real_gballoc_hl [](start = 10, length = 15)

not used (because the more complex mocking above)?

Reply:
yup, not used. Removed.

---
In reply to: [463689989](https://github.com/Azure/azure-c-util/pull/55#discussion_r463689989) [](ancestors = 463689989)